tests/contrib/funders/test_funders_api.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Funders API tests."""
+
+from functools import partial
+
+import pytest
+from invenio_indexer.api import RecordIndexer
+from invenio_search import current_search_client
+from jsonschema import ValidationError as SchemaValidationError
```

Comment:
question: I know I did this rename in `names` (According to git blame 😅) but I do not see the reason why... In some cases it might clash with `Marshmallow.ValidationError` but is not the case here... WDYT?

Reply:
I agree, while testing the API it won't probably clash with any `Marshmallow` exception since `Marshmallow` it is not used at this level (e.g. we're not (de)serialising data).